doc/api/https.md

```diff
@@ -533,7 +696,7 @@ const req = https.request(options, (res) => {
   console.log('All OK. Server matched our pinned cert or public key');
   console.log('statusCode:', res.statusCode);
   // Print the HPKP values
-  console.log('headers:', res.headers['public-key-pins']);
+  console.log('headers:', res.headers['strict-transport-security']);
```

Comment:
I don't understand this change. Can you explain?

Reply:
HPKP is obsolete so I guess GitHub no longer returns the header but in this case I think it is better to remove this instead of printing an unrelated header value.

Reply:
I'll do that!